packages/terra-form-select/src/search/Frame.jsx

```diff
@@ -528,18 +528,16 @@ class Frame extends React.Component {
    * Falls back to the string 'Search' if no label is provided
    */
   ariaLabel() {
-    const { ariaLabel, disabled, intl } = this.props;
-
+    const {
+      ariaLabel, disabled, intl, placeholder,
+    } = this.props;
     const defaultAriaLabel = intl.formatMessage({ id: 'Terra.form.select.ariaLabel' });
     const dimmed = intl.formatMessage({ id: 'Terra.form.select.dimmed' });
 
-    // VO on iOS doesn't do a good job of announcing disabled stated. Here we append the phrase that
-    // VO associates with disabled form controls.
-    if ('ontouchstart' in window && disabled) {
-      return ariaLabel === undefined ? `${defaultAriaLabel} ${dimmed}` : `${ariaLabel} ${dimmed}`;
-    }
+    const baseLabel = ariaLabel === undefined ? defaultAriaLabel : ariaLabel;
+    const modifiedAriaLabel = disabled && 'ontouchstart' in window ? `${baseLabel} ${dimmed}` : baseLabel;
 
-    return ariaLabel === undefined ? defaultAriaLabel : ariaLabel;
+    return placeholder ? `${modifiedAriaLabel}, ${placeholder}` : modifiedAriaLabel;
```

Comment:
After selection of an item, we still hear the placeholder text being announced along with the item selected when navigated back to searchselect component. Do we still want to hear this? 
single-select does not announce this way.

Reply:
We have removed placeholder text from aria label